src/features/feel-popup/FeelPopupManager.js

```diff
@@ -0,0 +1,56 @@
+export class FeelPopupManager {
```

Comment:
:spiral_notepad: This module is injected in the parent injector and can be overwritten. This particular one manages popup lifecycle heedlessly, and fires events for the renderer to hook into.